websites/M/MyAnimeList.Vip/dist/metadata.json

```
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.4",
+  "service": "MyAnimeList.Vip",
```

```suggestion
  "service": "MyAnimeList.vip",
```
I would do lowercase "v"...